example/ExampleDeviceClasses.h

```diff
@@ -1,12 +1,17 @@
 #pragma once
 
-class Alpha {
+class AlphaBetaParent {
+public:
+   virtual void Talk() = 0;
```

Comment:
this looks a bit like an interface, huh? :)
destructor with an empty definition would be good